README.md

```diff
@@ -1,5 +1,5 @@
 # Uno Toolkit
-A set of custom controls for the UWP, WinUI and the Uno Platform not offered out of the box by WinUI, such as Card, TabBar, NavigationBar, etc.
+A set of custom controls for the UWP, WinUI, and the Uno Platform not offered out of the box by WinUI, such as Card, TabBar, NavigationBar, etc.
```

Comment:
I'm wondering if we should start to phase out the use of UWP and just stick with WinUI going forward?

Reply:
```suggestion
Uno Toolkit provides a set of higher-level UI Controls designed specifically for multi-platform, responsive applications.
```

Reply:
Thanks @jeromelaban !